.gitlab-ci.yml

```diff
@@ -52,6 +52,25 @@ build:
     paths:
       - image.tar
 
+test:format:
+  stage: test
+  needs: []
+  image: python:3
+  before_script:
+    - pip install beautysh
+  script:
+    # Core script
+    - beautysh --indent-size 2 mender-convert*
```

Comment:
I guess this becomes more of a vote, but could we make it `4` instead of `2` to match Python? Otherwise this becomes the third indentation style (Golang-Tabs, Python-4spc) in our projects. It is also especially relevant in Yocto context, where shell and Python are mixed in the same file (not that I expect it to handle this, but I'm talking about our policy).

I haven't checked explicitly, but I suspect we have more 4-space scripts than 2-space scripts in general. Mender-convert seems to have a bit of both.

Reply:
I am in favor for 4 spaces. Not only for the reasons you mentioned above, but because it is the default in `beautysh` and there is no way to configure it via config file -> Meaning, every time you run it manually you need to remember this cli parameter.

This will however modify virtually all the code of the core scripts in this repository. Are we sure about it?

Reply:
Just to add here: When we started out, `mender-convert` was supposed to follow the guidelines, as laid out by google's [shell-guide](https://google.github.io/styleguide/shellguide.html#s5.1-indentation). Which is why `2` was chosen. 

I would personally choose to stay with the `2` spaces which is already here. Both for consistency with the style already here, and because I _personally_ think `4` is a little verbose for shell scripts. The argument to beatush isn't really an issue imo

Reply:
I think so. The opposite would modify most files in some other repository, so I think it's a catch-22. I'm happy to bring it up to a larger vote though.

Reply:
Okey. One voice against a bit change is enough to block it.

Good points, by the way. Thanks!

Reply:
Also, I see [this](https://github.com/mvdan/sh) is a pretty popular shell-formatter, as compared to `beautysh`. Should we consider it?